### PR TITLE
add limit

### LINKED ISSUE #9: Add "limit" method
The `str.limit(limit, end = '…')` method returns the first `limit` characters and ends the limited string with `end`.

**Example**

```js
Str('Supercharge').limit(5, ' ->').get()

// 'Super ->'
```

### PR DESCRIPTION
**Pull request by mcvladthegoat**
Added `limit` feature
Closes  #9

### CODE REVIEW

**Comment by marcuspoehls** [comment]
@mcvladthegoat Hey Vlad, thank you for the PR. Looks good!

While reviewing the code, I was thinking about the case where the original string is shorter than the limit. Let's say you want to limit have character limit of 100. The string going into the method has 60 characters. In such cases, I think we should not append the `end`. What do you think? 

Example:

```js
Str('Short string').limit(100, '…').get()
// 'Short string' 
// or
// 'Short string…'
```

File: src/str.js [hunk 1]
```diff
@@ -204,6 +204,21 @@ class Str {
...
+   *
+   *
+   * @param {Number} limit
+   * @param {String} end
+   *
+   * @returns {Str}
+   */
+  limit (limit = 0, end = '') {
+    return new Str(
+      this.value.substr(0, limit).concat(end)
```
Comment (marcuspoehls) [inline src/str.js:218]: Can you please use the `.substring()` method? The `substr` method is deprecated.

Comment (mcvladthegoat) [reply src/str.js]: Ok, switched to `substring`

Comment (marcuspoehls) [reply src/str.js]: Thank you!

**Comment by mcvladthegoat** [comment]
@marcuspoehls oh, yes, i agree with you, it shouldn't end with `...` in that case. I'll change it and commit asap

**Comment by marcuspoehls** [comment]
@mcvladthegoat thank you

**Comment by marcuspoehls** [comment]
@mcvladthegoat Thank you Vlad, looks good

I noticed the details whether to limit the string on length only after seeing your implementation and tests. Thank you for your help and work on this. I really appreciate it!

**Pull request merged**